native/go/Makefile

```diff
@@ -19,8 +19,10 @@ include ../../mk/spksrc.native-cc.mk
 
 myComp:
 	@$(MSG) "Building Go"
-	cd $(WORK_DIR)/$(PKG_NAME)/src && ./make.bash
+	cd $(WORK_DIR)/$(PKG_NAME)/src && GOROOT_BOOTSTRAP=$(WORK_DIR)/../../go-1.4/work-native/go ./make.bash
 	@$(MSG) "Building Go for arm"
-	cd $(WORK_DIR)/$(PKG_NAME)/src && export GOARCH=arm && export GOARM=5 && ./make.bash --no-clean
+	cd $(WORK_DIR)/$(PKG_NAME)/src && export GOARCH=arm && export GOARM=5 && GOROOT_BOOTSTRAP=$(WORK_DIR)/../../go-1.4/work-native/go ./make.bash --no-clean
 	@$(MSG) "Building Go for 386"
-	cd $(WORK_DIR)/$(PKG_NAME)/src && export GOARCH=386 && ./make.bash --no-clean
+	cd $(WORK_DIR)/$(PKG_NAME)/src && export GOARCH=386 && GOROOT_BOOTSTRAP=$(WORK_DIR)/../../go-1.4/work-native/go ./make.bash --no-clean
+	@$(MSG) "Building Go for amd64"
```

Comment:
You might have noticed that the build output at some point looks like this:
```
===>  Building Go for 386
##### Building Go bootstrap tool.
##### Building Go toolchain using /home/bean/spksrc/native/go/work-native/../../go-1.4/work-native/go.
##### Building go_bootstrap for host, linux/amd64.
##### Building packages and commands for host, linux/amd64.
##### Building packages and commands for linux/386.
```
Essentially, every time you build for a specific `GOARCH`, Go starts from the beginning: it checks if there's a bootstrap available for the host, then builds the actual `go` command for host (which is amd64 for spksrc), and then builds the command for the specified `GOARCH`.
That means that L27/28 are not needed: host/amd64 is already built at that point: you actually start off with it on L22.